flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
it's open source but you only package the Intel binary.

great sadness.

Reply:
I am not aware of a amd64 arch...

Reply:
`aarch64` is the one I'm talking about.

Reply:
this package has no arm version. Only x64 and x86_64

Reply:
How about building from source ? The Electron runtime do exist for `aarch64`

BTW x64 and x86_64 are the exact same thing. The former is just Microsoft, the latter being the used everywhere else (Debian excepted as they use amd64)

Reply:
They have no npm task for aarch64, and I dont know how to build a electron app outside of running npm tasks. 

Maybe we should ask the Rambox team to make a proper Implemention so non flatpak versions can be made for arm  as well and we can be sure there no problems.

Reply:
In author's defence, this might be open source, but it has some [peculiar build requirements](https://github.com/ramboxapp/community-edition/blob/master/CONTRIBUTING.md#prerequisites) including what seems to be a proprietary program only shipped for x86.  Furthermore, and perhaps most importantly, it would seem that when you build it yourself, you have to go and get your own Auth0 API key.  Unpacking the upstream build doesn't seem so unreasonable given all that.

Reply:
@hfiguiere Building electron apps from source is _hard_, maintaining them afterwards is even harders.
In this particular app, I don't even see a lockfile in the source repo, meaning we'll have even more work maintaining it and can't make deterministic builds.

Reply:
Yeah. When I tried it I had to generate lockfile. We would have to regenerate the lockfile every single update. Automating this would be very  hard, on top having to manage it in general.

I don't think its worth the effort.